### PR TITLE
🧪 Created client side experiment for page outlink

### PR DESCRIPTION
**Pull request by jshamble**
Created client side experiment to accompany [the CTA migration page outlink layer](https://github.com/ampproject/amphtml/pull/35983)

⚠️Do not merge until experiment/beta is launched.⚠️

Part of https://github.com/ampproject/amphtml/issues/36035

### CODE REVIEW

File: build-system/global-configs/client-side-experiments-config.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
     {
       "name": "story-load-inactive-outside-viewport",
       "percentage": 1
+    },
+    {
+      "name": "story-ad-page-outlink",
+      "percentage": 1
```
Comment (calebcordry) [inline build-system/global-configs/client-side-experiments-config.json:29]: Let's start with beta only. Also we need to make sure not to merge this before the changes are live (should be Wednesday).

**Review by powerivq — APPROVED** [review]

**Comment by danielrozenberg** [comment]
`03` is the prefix for beta, `00` is the prefix for experimental - each of them receive 0.5% traffic and are always the same version. You might want to add `00` to the prefix list, but that's up to you!

**Review by danielrozenberg — APPROVED** [review]

**Comment by calebcordry** [comment]
> `03` is the prefix for beta, `00` is the prefix for experimental - each of them receive 0.5% traffic and are always the same version. You might want to add `00` to the prefix list, but that's up to you!

Good idea, let's also enable experimental. Please make sure that percentage promote is complete before merging, right now its only in opt-in.

File: build-system/global-configs/client-side-experiments-config.json (hunk 1 above)
Comment (jshamble) [reply build-system/global-configs/client-side-experiments-config.json]: Ok, sounds good, I'm starting with Beta only.

**Review by calebcordry — APPROVED** [review]

File: build-system/global-configs/client-side-experiments-config.json [hunk 2]
```diff
@@ -23,6 +23,18 @@
     {
       "name": "story-load-inactive-outside-viewport",
       "percentage": 1
+    },
+    {
+      "name": "amp-sticky-ad-to-amp-ad-v2",
```
Comment (jridgewell) [inline build-system/global-configs/client-side-experiments-config.json:28]: Why is this added? Bad rebase?

**Review by jridgewell — CHANGES_REQUESTED** [review]

File: build-system/global-configs/client-side-experiments-config.json [hunk 3]
```diff
@@ -23,6 +23,18 @@
...
+      "percentage": 0.05
+    },
+    {
+      "name": "story-ad-page-outlink",
+      "percentage": 1,
```
Comment (jridgewell) [inline build-system/global-configs/client-side-experiments-config.json:33]: Is your intention to launch this, or to experiment?

Comment (jshamble) [reply build-system/global-configs/client-side-experiments-config.json:33]: Experiment first with this PR, (then eventually launch).

File: build-system/global-configs/client-side-experiments-config.json (hunk 2 above)
Comment (jshamble) [reply build-system/global-configs/client-side-experiments-config.json:28]: I'll look into it - yes, probably a bad rebase.

**Review by jridgewell — APPROVED** [review]
